### PR TITLE
Adds Bricks and Rubble

### PR DESCRIPTION
**Pull request by AcapitalA**
Bricks and rubble have been added to Deltastation's maintenance northeast of atmospherics...
hmm

### CODE REVIEW

**Comment by AcapitalA** [comment]
DNM possible shitlery going on.

**Comment by metroid1310** [comment]
die

**Comment by AcapitalA** [comment]
> die

Thanks for the feedback kind stranger!

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -101026,6 +101061,13 @@
 	},
 /turf/open/floor/iron,
 /area/security/office)
+"uVT" = (
+/obj/item/clothing/under/color/red,
+/obj/effect/decal/remains/human,
+/turf/open/floor/plating,
+/area/maintenance/port/fore{
+	desc = "How did...   What the hell?"
+	})
```
Comment (TheChosenEvilOne) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]:
```suggestion
/area/maintenance/port/fore)
```

Comment (TheChosenEvilOne) [reply _maps/map_files/Deltastation/DeltaStation2.dmm]: can't edit vars with /areas

Comment (AcapitalA) [reply _maps/map_files/Deltastation/DeltaStation2.dmm]: I remember removing that desc, weird that it saved that. Looking into it.

**Pull request merged**